src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -133,6 +133,20 @@
       help = "If non-empty read the specified resolved file instead of the WORKSPACE file")
   public String experimentalResolvedFileInsteadOfWorkspace;
 
+  @Option(
+    name = "experimental_remote_downloader_config",
+    defaultValue = "null",
+    documentationCategory = OptionDocumentationCategory.REMOTE,
+    effectTags = {OptionEffectTag.UNKNOWN},
+    help =
+      "Specify a file to configure the remote downloader with. This file consists of lines, "
+        + "each of which starts with a directive (`allow`, `block` or `rewrite`) followed "
+        + "by either a host name (for `allow` and `block`) or two patterns, one to match "
+        + "against, and one to use as a substitute URL, with back-references starting from "
+        + "`$1`. It is possible for multiple `rewrite` directives for the same URL to be "
+        + "give, and in this case multiple URLs will be returned.")
+  public String remoteDownloaderConfig;
```

Comment:
Can we remove the `remote` prefix? This is confused with `RemoteDownloader` in remote module.